src/main/scala/org/hammerlab/guacamole/reference/package.scala

```diff
@@ -0,0 +1,5 @@
+package org.hammerlab.guacamole
+
+package object reference {
+  type ContigSequence = Seq[Byte]
```

Comment:
I like the idea of making it an alias for `IndexedSeq`! Wrapping it in another class (or trait, and then subclassing that) feels like added overhead for no benefit.